r/redisearch/Dockerfiles/2.0.0-ubi8/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM registry.access.redhat.com/ubi8/ubi AS builder
```

Comment:
let's explicitly add an attribution stating that this Dockerfile is based on and derived from the community versions at <GitHub URL>

Reply:
Changes made as per your suggestion.